### ISSUE TITLE
Working with React 16.13.0

### ISSUE THREAD

**Issue by xeusteerapat** [prabhuignoto/react-chrono#42]
I ran to the error about `Invalid hook call` when created the project with React 16.13.0

```
Error
Invalid hook call. Hooks can only be called inside of the body of a function component. This could happen for one of the following reasons:
1. You might have mismatching versions of React and the renderer (such as React DOM)
2. You might be breaking the Rules of Hooks
3. You might have more than one copy of React in the same app
See https://reactjs.org/link/invalid-hook-call for tips about how to debug and fix this problem.
```

How to solve this issue? or we just need to use React 17.0.0 or the newest?

**Comment by prabhuignoto** [comment]
Please upgrade to 17 and the error should go away. let me know how it pans out.

**Comment by xeusteerapat** [comment]
Yeah, there is no error when I've upgraded to 17.0.0 but kinda curious is any possible to use 16.x.x version?

**Comment by prabhuignoto** [comment]
sure will look into what is causing this.

**Comment by prabhuignoto** [comment]
@xeusteerapat just published the new version 1.2.1. should work for 16 and 17 now.

**Comment by xeusteerapat** [comment]
Hi, there is no error like the previous one after upgraded to the latest version.
